pkg/config/config.go

```diff
@@ -18,5 +18,7 @@ const (
 	// OCMClientSecret represents the OCM Client ID that will be used for testing fleet-mode run
 	OCMClientSecret string = "ocm-client-secret" //#nosec G101 -- This is a false positive
 
-	ServiceLogService string = "service_logs"
+	ServiceLogService string = "service_logs" //#nosec G101 -- This is a false positive
+
+	ClustersService string = "clusters" //#nosec G101 -- This is a false positive
```

Comment:
This should be `clusters_service`? to match the ocm service name

Reply:
I think it should actually be `clusters_mgmt` to be in line with api.openshift.com, but good pickup :+1: :
![image](https://github.com/openshift/ocm-agent/assets/28124379/37ab510e-fe95-44ee-99c6-91046fad3117)